history/models.py

```diff
@@ -563,3 +563,12 @@ def predict(self, sample):
         recommend = self.recommend_trade(nn_price, last_sample)
         projected_change_pct = (nn_price - last_sample) / last_sample
         return recommend, nn_price, last_sample, projected_change_pct
+
+
+class SocialNetworkMention(AbstractedTesterClass):
+    network_name = models.CharField(max_length=30, db_index=True)
+    network_username = models.CharField(max_length=100)
+    network_id = models.BigIntegerField(default=0)
+    network_created_on = models.DateTimeField(default=get_time)
```

Comment:
This should be `auto_now_add=True` instead of default. `get_time` is not UTC and I left it there only because it's used in an old migration.